docker_shell.sh

```diff
@@ -90,7 +103,7 @@ docker run --name "bazel-orfs-$uuid" --rm \
  bash -c \
  "set -ex
  . ./env.sh
- cd \$FLOW_HOME
+ cd \$BUILD_DIR
```

Comment:
This is missing from the generated  `_make` scripts, near as I can see

Reply:
Turns out that with most recent modifications it was actually beneficial to also do that in `orfs`